Functions.Templates/Templates/CosmosDBTrigger-CSharp/build.config/vs-2017.3.host.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": {
+    "text": "CosmosDBTrigger",
+    "package": "{E9BCFE2B-304A-4D7F-836F-F3A4FB1A1451}",
+    "id": "1034"
+  },
+  "description": {
+    "text": "A C# function that will be run whenever documents change in a document collection.",
+    "package": "{E9BCFE2B-304A-4D7F-836F-F3A4FB1A1451}",
+    "id": "1035"
+  },
+  "order": 100,
+  "icon": "vs-2017.3/function_f.png",
+  "learnMoreLink": "https://go.microsoft.com/fwlink/?linkid=848145",
+  "uiFilters": [
+    "functions"
+  ]
```

Comment:
Add symbolInfo for each of the symbol in the template.json file. 
Here is a sample
https://github.com/Azure/azure-webjobs-sdk-templates/blob/dev/Functions.Templates/Templates/ServiceBusQueueTrigger-CSharp/build.config/vs-2017.3.host.json#L18-L46.